contracts/token/ERC20/ERC20Permit.sol

```diff
@@ -0,0 +1,178 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./ERC20.sol";
```

Comment:
Inherit from `ERC20Base`.

Reply:
I had to also add inheritance from `ERC20Metadata`
Do we keep inheritance from both `ERC20Base` + `ERC20Metadata` or do I revert to only inheritance from `ERC20` ?

Reply:
Seems strange that `name()` is needed for this, but if that's the case then `ER20Base` + `ERC20Metadata` inheritance is correct.